Protocol.md

```diff
@@ -107,7 +107,7 @@ connection.
 
 A SaltyRTC complicant server SHOULD have at least one permanent NaCl key
 pair for public key authenticated encryption. If the server has such a
-key pair, it will be used to sign<sup>1</sup> the server's session key
+key pair, it will be used to sign¹ the server's session key
```

Comment:
Keeping `<sup>` is a good idea I believe.

Reply:
Any reason? This is much easier to read (and grep).

Reply:
Mh, no particular reason other than it seems to be the way to go in Markdown.

Reply:
Well, it's just HTML. But you wouldn't start to write `&ouml;` in Markdown instead of `ö`, even though `&ouml;` is "the way to go" for ASCII based Markdown files ;)